c2corg_ui/templates/waypoint/helpers/view.html

```diff
@@ -423,3 +423,27 @@
     </div>
   % endif
 </%def>
+
+<%def name="get_waypoint_access_field(locale, waypoint_type, open_tab=False)">\
+  % if locale.get('access'):
+    <%
+        if waypoint_type == 'access':
+            attribute_name = 'pt access'
+        elif waypoint_type in ['hut', 'climbing_indoor', 'climbing_outdoor']:
+            attribute_name = 'pedestrian access'
+        else:
+            attribute_name = 'road or pedestrian access'
```

Comment:
Again this is not a very "safe" way to switch among attribute titles. It's a bit more verbose but clearer (and at least you are sure the strings will be i18n-extracted) to use 
```
% if waypoint_type == 'access':
  <span x-translate>pt access</span>
...
```